src/Ssh.php

```diff
@@ -111,7 +111,7 @@ protected function getExtraOptions(): string
             $extraOptions[] = "-i {$this->pathToPrivateKey}";
         }
 
-        if ($this->port) {
+        if ($this->port > -1) {
```

Comment:
I'm thinking that this code now allows the invalid port `-1` to be used.

Reply:
This works fine (https://3v4l.org/ufUIW) but I would agree that `>= 0` reads better than `-1` in this context

Reply:
> This works fine

Ah, right I was mistaken. Yeah, best change it to `>=` for better readability. Thanks!

Reply:
comparison changed to `>= 0`